argus.js

```diff
@@ -2,7 +2,7 @@ const MESSAGE_ARGUS_TOKEN_REFRESH = "argus-token-refresh";
 const MESSAGE_ARGUS_TOKEN_REQUEST = "argus-token-request";
 const MESSAGE_ARGUS_TOKEN_RESPONSE = "argus-token-response";
 
-const ArgusJS = function(token) {
+const ArgusJS = function(token, mdrUrl) {
```

Comment:
You also need to update line 30 to include the URL:

`resolve(ArgusJS(event.data.token));` -> `resolve(ArgusJS(event.data.token, event.data.mdr_url));`